woocommerce/woocommerce
# Add navigation and progress-bar components for the new core profiler

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:



Closes https://github.com/woocommerce/team-ghidorah/issues/193

This PR adds 'navigation' component that provides navigation UI for the new core profiler.

The component was copied from WPCOM and converted to TS with a few modifications.





### How to test the changes in this Pull Request:

1. Review the test and make sure it makes sense.
2. Try to use the component in one of the pages and confirm it renders as expected
